k8s/airflow/Makefile

```diff
@@ -0,0 +1,68 @@
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+include ../images.Makefile
+
+CHART_NAME := airflow
+APP_ID ?= $(CHART_NAME)
+WAIT_FOR_READY_TIMEOUT := 600
+
+TRACK ?= 2.2
+# Track for components
+POSTGRESQL_TRACK ?= 13.4
+NFS_TRACK ?= 1.3
+METRICS_EXPORTER_TAG ?= v0.5.1
+
+VERIFY_WAIT_TIMEOUT = 1800
+
+SOURCE_REGISTRY ?= marketplace.gcr.io/google
+
+IMAGE_AIRFLOW ?= $(SOURCE_REGISTRY)/airflow2:$(TRACK)
+
+# POSTGRESQL
+IMAGE_POSTGRESQL ?= $(SOURCE_REGISTRY)/postgresql:$(POSTGRESQL_TRACK)
```

Comment:
`gcr.io/cloud-marketplace/google/postgresql13`

Reply:
thank you, fixed